incubator/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -1210,6 +1213,9 @@ private final class MqttServer
         private final Int2ObjectHashMap<String> topicAliases;
         private final Int2IntHashMap subscribePacketIds;
         private final Object2IntHashMap<String> unsubscribePacketIds;
+        private final GuardHandler guard;
+        private final Function<String, String> credentials;
+        private final String authFieldName;
```

Comment:
This will become the same `enum` as mentioned above for `MqttCredentialsConfig`.